README.md

```diff
@@ -38,11 +37,19 @@ export default LambdaWrapper({}, (di, request, done) => {
 
 ## Serverless Offline & SQS Emulation
 
-Serverless Offline only emulates API Gateway and Lambda, so publishing an SQS message would use the real SQS queue and trigger the consumer function (if any) in AWS. When working with offline code, the expectation is that the local functions will be invoked instead.
+Serverless Offline only emulates API Gateway and Lambda, so publishing an SQS message would use the real SQS queue and trigger the consumer function (if any) in AWS. When working with offline code, you often want the local functions to be invoked instead.
 
-In order to emulate SQS, `SQSService.prototype.publish` will check `DependencyInjection.prototype.isOffline`. If it's `true`, a Lambda client will be created and the message will be delivered to the offline Lambda endpoint, effectively running the consumer function _immediately_ as part of the original Lambda invocation. This works very well in the offline environment because invoking a Lambda function will trigger its whole (local) execution tree.
+Offline SQS behaviour can be configured by setting the `LAMBDA_WRAPPER_OFFLINE_SQS_MODE` environment variable. Available modes are:
 
-### How to use it?
+- `lambda` (the default): send messages to an offline Lambda endpoint
+- `sqs`: send messages to an offline SQS service, such as Localstack
```

Comment:
If I am not mistaken, this is done by sending a HTTP request to a localhost port, right? So maybe `http` makes sense here?

```suggestion
- `http`: send messages to an offline SQS service, such as Localstack
```

Reply:
How about `default` as it seems like a natural outcome for an offline mode of an SQS service? I fear `http` raises expectations that this mode might do something else.

Reply:
Right, it makes an HTTP request to a localhost SQS endpoint. However the same is true for the lambda mode; the consumer function invocation is done by HTTP request to a localhost Lambda endpoint. So for me `http` is not distinct enough.

Similarly `offline` could describe both the offline Lambda and offline SQS modes.

My original names were motivated by "what offline service will handle SQS messages?", so maybe part of the solution would be to update the environment variable name (and docs) to reflect this, if you agree with that reasoning.

Otherwise what do you think about these?

- `lambda` or `direct` – invoke the consumer directly instead of publishing to SQS
- `local` – use a local SQS endpoint
- `aws` – send to AWS as normal

Reply:
I spent so long staring at my comments by the time I'd pressed 'add' things had moved on. I like your suggestion here.

Reply:
Agree with the new proposed names (`direct` sounds better than `lambda`)